src/serac/numerics/functional/tests/functional_basic_h1_vector.cpp

```diff
@@ -138,20 +162,26 @@ void test_suite(std::string meshfile)
   }
 }
 
+TEST(VectorValuedH1, test_suite_hexes) { test_suite("/data/meshes/patch3D_hexes.mesh"); }
+
+#ifndef SERAC_USE_CUDA_KERNEL_EVALUATION
 TEST(VectorValuedH1, test_suite_tris) { test_suite("/data/meshes/patch2D_tris.mesh"); }
 TEST(VectorValuedH1, test_suite_quads) { test_suite("/data/meshes/patch2D_quads.mesh"); }
 TEST(VectorValuedH1, test_suite_tris_and_quads) { test_suite("/data/meshes/patch2D_tris_and_quads.mesh"); }
-
 TEST(VectorValuedH1, test_suite_tets) { test_suite("/data/meshes/patch3D_tets.mesh"); }
-TEST(VectorValuedH1, test_suite_hexes) { test_suite("/data/meshes/patch3D_hexes.mesh"); }
 TEST(VectorValuedH1, test_suite_tets_and_hexes) { test_suite("/data/meshes/patch3D_tets_and_hexes.mesh"); }
+#endif
 
 int main(int argc, char* argv[])
 {
   int num_procs, myid;
 
   ::testing::InitGoogleTest(&argc, argv);
 
+#ifdef SERAC_USE_CUDA_KERNEL_EVALUATION
+  serac::accelerator::initializeDevice();
+#endif
```

Comment:
It would be better if there was just a call to:
```c++
  ::serac::initialize();
```
that happens all the time (regardless of CPU, GPU, APU, etc.) rather than having the #ifdef USE_CUDA, #ifdef USE_HIP, etc.